presentation/courses/addCourse.php

```diff
@@ -50,51 +51,22 @@
 <h2>Add Course</h2>
 <form method="post" action="../../data/courses/addCourseScript.php">
     <div>
-        <p>Course ID
-            <input name="course_code" type="text" size="10" required/><br/></p>
-        <p>Course Title
-            <input name="title" type="text" size="40" required/>
-        </p><br/>
+        <p>Student ID
+            <label>
+                <input name="id" type="number" size="10" placeholder="Student ID" required/>
```

Comment:
Able to toggle integer values (strange) and also insert negative values as well (bad):

![image](https://user-images.githubusercontent.com/48603412/194760291-be2c0f55-426b-4a06-8550-eb223d1894a7.png)

Reply:
Will check for negative integeters, but other suggestions are optional features for later, but good idea. Will push PR soon.